### DISCUSSION TITLE
Asymmetrical  airfoil

### DISCUSSION THREAD

**Discussion by Janfoam** [mdolab/dafoam#147] [Tutorials]
Hello, 

I have an asymmetrical airfoil case that I am running and could use some tips about best practices for the FFD box.  Because of the asymmetrical nature of the airfoil the suction side is closer to the FFD box than the pressure side.  Would it be best to ensure the pressure side is closer or this has very little effect?

![image](https://user-images.githubusercontent.com/76228791/121740193-0ab36e00-cafd-11eb-8f65-2fa8bea3ebd1.png)

**Comment by friedenhe** [comment]
The FFD points look good. It is NOT necessary to make the airfoil in the very middle of the FFD box.
